### PR TITLE
[MSGraphPS] Feedback about client secret credentials

### PR DESCRIPTION
**Pull request by csmulligan**
[User story](https://dev.azure.com/msft-skilling/Content/_workitems/edit/238773)

### CODE REVIEW

File: microsoftgraph/docs-conceptual/authentication-commands.md [hunk 1]
```diff
@@ -97,7 +97,9 @@ To use app-only access, you can load the certificate from either *Cert:\CurrentU
 This type of grant will help when you need interactions in the background without a user to sign in. Support for client secret credentials was added by adding **-ClientSecretCredential** parameter to **Connect-MgGraph**. See [Get-Credential](/powershell/module/microsoft.powershell.security/get-credential) on how to get or create credentials.
 
 ```powershell
-$ClientSecretCredential = Get-Credential -Credential "Client_Id"
+$User = <Application (client) ID>
+$PWord = ConvertTo-SecureString -String <Client Secret> -AsPlainText -Force
+$ClientSecretCredential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $User, $PWord
```
Comment (csmulligan) [inline microsoftgraph/docs-conceptual/authentication-commands.md:102]: Hi @SteveMutungi254 a customer suggested this script in a GitHub issue, stating that the one in our documentation didn’t work. Could you please check if this is correct?
Thank you in advance!

Comment (SteveMutungi254) [inline microsoftgraph/docs-conceptual/authentication-commands.md:102]:
```suggestion
# Define the Application (Client) ID and Secret
$ApplicationClientId = '<application(client)ID>' # Application (Client) ID
$ApplicationClientSecret = '<secret.value>' # Application Secret Value
$TenantId = 'Tenant_Id' # Tenant ID

# Convert the Client Secret to a Secure String
$SecureClientSecret = ConvertTo-SecureString -String $ApplicationClientSecret -AsPlainText -Force

# Create a PSCredential Object Using the Client ID and Secure Client Secret
$ClientSecretCredential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $ApplicationClientId, $SecureClientSecret
```

File: microsoftgraph/docs-conceptual/authentication-commands.md [hunk 2]
```diff
@@ -97,7 +97,9 @@ To use app-only access, you can load the certificate from either *Cert:\CurrentU
...
 # Enter client_secret in the password prompt.
 Connect-MgGraph -TenantId "Tenant_Id" -ClientSecretCredential $ClientSecretCredential
```
Comment (SteveMutungi254) [inline microsoftgraph/docs-conceptual/authentication-commands.md:104]:
```suggestion
# Connect to Microsoft Graph Using the Tenant ID and Client Secret Credential
Connect-MgGraph -TenantId $TenantId -ClientSecretCredential $ClientSecretCredential
```

Comment (csmulligan) [reply microsoftgraph/docs-conceptual/authentication-commands.md]: Thank you @SteveMutungi254!

**Comment by csmulligan** [comment]
Hi @timayabi2020 could you please double-check the changes in this PR? Many thanks!

**Review by timayabi2020 — APPROVED** [review]

**Pull request merged**